dipy/tracking/utils.py

```diff
@@ -503,6 +515,11 @@ def random_seeds_from_mask(mask, seeds_per_voxel=1, affine=None):
     seeds = where + grid - .5
     seeds = asarray(seeds)
 
+    if not seed_count_per_voxel:
+        # Randomize the seeds and select the requested amount
+        np.random.shuffle(seeds)
```

Comment:
My view is that there is no reason to have a parameter for the random state. It will bring more confusion than good.  But if you add in the documentation that you can actually specify the randomness (using np.random.seed...) before calling the function that would be enough.